### ISSUE TITLE
Slack team for IIITV

### ISSUE THREAD

**Issue by aviaryan** [iiitv/ShareIt#11]

**Comment by aviaryan** [comment]
Signup from https://iiitvadodara.slack.com/signup using college email address.

**Comment by arpit1997** [comment]
Is it different from trello ?

**Comment by harishkrupo** [comment]
is it like gitter?

**Comment by aviaryan** [comment]
@harishkrupo yes but it is much more feature-rich, polished and popular. You can write bots and integrate it with all sorts of services.

@arpit1997 Slack is slack and Trello is trello. Both are way different and the best in their class.
